phys2bids/physio_obj.py

```diff
@@ -469,7 +469,12 @@ def check_trigger_amount(self, thr=None, num_timepoints_expected=0, tr=0):
         # Check if thr was given, if not "guess" it.
         flag = 0
         if thr is None:
-            thr = np.mean(trigger) + 2 * np.std(trigger)
+            # If trigger channels are binary 
+            # (i.e., "on" is a higher value and "off" is a lower value)
+            # and each "on" and "off" are each always approzimately the same value
+            # then any value above the mean is "on" and every value below the mean
+            # is "off". 
+            thr = np.mean(trigger)
```

Comment:
I see your point, although it happens that the trigger values oscillate in the off and on states so there is no plateau. However,  If you think that will not affect the detection, go for it

Reply:
My thinking is that the magnitude of the oscillations about "on" and "off" are unlikely to be so large that they approach the mean. Would be happy to try this out with a few different datasets to test that theory!

Reply:
You are probably right